packages/admin-next/dashboard/src/v2-routes/invite/invite.tsx

```diff
@@ -232,8 +232,8 @@ const CreateView = ({
         description: t("invite.toast.accepted"),
         dismissLabel: t("actions.close"),
       })
-    } catch (error) {
-      if (isAxiosError(error) && error.response?.status === 400) {
+    } catch (error: any) {
+      if ("type" in error && error.type === "invalid_data") {
```

Comment:
@olivermrbl I might be wrong but other places are showing a toast with the error message, should I do that?

Reply:
I don't think we should if the form is showing the error. Is it doing that?

Reply:
No it does not because there is no response on the error, there is the type, message and no code as well

Reply:
here the error I was playing with is duplicated email